tracks/pep8/lab1-1/tests.txt

```diff
@@ -0,0 +1,16 @@
+===
```

Comment:
I'm really not sure about one file for all tests.

Maybe this can be an option? one file `tests.*` or a dir `tests/` ?